pkg/testutils/config.go

```diff
@@ -13,18 +13,29 @@ const BLOCKCHAIN_RPC_URL = "http://localhost:7545"
 // This is safe to commit
 const TEST_PRIVATE_KEY = "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
 
+const (
+	RATE_REGISTRY_ADDRESS             = "0xE71ac6dE80392495eB52FB1dCa321f5dB8f51BAE"
+	NODE_REGISTRY_ADDRESS             = "0x8d69E9834f1e4b38443C638956F7D81CD04eBB2F"
+	GROUP_MESSAGE_BROADCAST_ADDRESS   = "0xD5b7B43B0e31112fF99Bd5d5C4f6b828259bedDE"
+	IDENTITY_UPDATE_BROADCAST_ADDRESS = "0xe67104BC93003192ab78B797d120DBA6e9Ff4928"
+)
```

Comment:
we will have to figure out a better way to fetch these. Same for hardcoding the version of the contracts image. I will give it some thought and can create a followup PR